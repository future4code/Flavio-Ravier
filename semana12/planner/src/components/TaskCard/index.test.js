import React from 'react'
import {TaskCard} from './index'
import renderer from "react-test-renderer";

test('Snapshot', () => {
    const tree = renderer.create(<TaskCard />).toJSON();
    expect(tree).toMatchSnapshot();
})