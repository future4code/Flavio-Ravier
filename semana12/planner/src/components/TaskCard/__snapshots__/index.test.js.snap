// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<div
  className="MuiPaper-root sc-bdVaJa jSmAnv MuiPaper-elevation1 MuiPaper-rounded"
>
  <h5
    className="MuiTypography-root MuiTypography-overline MuiTypography-colorPrimary"
  />
</div>
`;
